1.16/Dockerfile

```diff
@@ -0,0 +1,92 @@
+FROM centos/s2i-core-centos7:1
+
+# RHSCL rh-nginx114 image.
+#
+# Volumes:
+#  * /var/opt/rh/rh-nginx114/log/nginx/ - Storage for logs
+
+EXPOSE 8080
+EXPOSE 8443
+
+ENV NAME=nginx \
+    NGINX_VERSION=1.14 \
```

Comment:
I'd rather use correct versions and if the packages are not available yet, let the container build fail.